src/core/agent/services/multiSigService.ts

```diff
@@ -571,13 +575,11 @@ class MultiSigService extends AgentService {
       rot: [serder, atc],
     };
 
-    const smidsValues = smidsMember.map((item) => item.i);
-    const rmidsValues = rmidsMember.map((item) => item.i);
-    const recp = [...new Set([...smidsValues, ...rmidsValues])];
-
-    const rstates = [
-      ...smidsMember,
-      ...rmidsMember.filter((rm) => !smidsMember.some((sm) => sm.i === rm.i)),
+    const recp = [
+      ...new Set([
+        ...smids.map((item) => item.aid),
```

Comment:
Ugh, isn't `smids` just `members?.signing`, which is a list of strings?

So `[...new Set([...smids, ...rmids])]`

Reply:
nevermind, seems not